lib/exact_target_sdk/client.rb

```diff
@@ -328,6 +328,7 @@ def soap_instance_url
   end
 
   def request_access_token
+    return if @access_token = Rails.cache.fetch(access_token_cache_key(config[:username]), nil)
```

Comment:
Gem should be framework agnostic. I expect the caching to be done in the product part.

Reply:
So we need to be able to pass cached access_token in the library.